FirebaseAdmin/FirebaseAdmin/Messaging/ErrorInfo.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+
+namespace FirebaseAdmin.Messaging
+{
+    /// <summary>
+    /// A topic management error.
+    /// </summary>
+    public sealed class ErrorInfo
+    {
+        // Server error codes as defined in https://developers.google.com/instance-id/reference/server
+        // TODO: Should we handle other error codes here (e.g. PERMISSION_DENIED)?
+        private static IReadOnlyDictionary<string, string> errorCodes;
```

Comment:
Let's assign this dictionary here (i.e. move that logic out of the constructor). Also make it `readonly`.